controller-server/src/main/java/com/yahoo/vespa/hosted/controller/persistence/ApplicationSerializer.java

```diff
@@ -275,13 +276,15 @@ private void jobRunToSlime(JobStatus.JobRun jobRun, Cursor parent, String jobRun
     }
 
     private void toSlime(Change deploying, Cursor parentObject, String fieldName) {
-        if ( ! deploying.isPresent()) return;
+        if ( ! deploying.isPresent() && ! deploying.isPinning()) return;
```

Comment:
Can `Change#isPresent` be updated to return `true` if pin is set? Or in other words, can a change only be considered empty if it's unpinned and platform and app version are unset?

Reply:
That doesn't fit well with the usage (and doc) for `Change#isPresent`.